### PR TITLE
store/mgostore: use mgotest for mongodb testing

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/146/

File: store/testing/meeting.go [hunk 1]
```diff
@@ -34,7 +34,9 @@ func (s *MeetingSuite) SetUpTest(c *gc.C) {
 }
 
 func (s *MeetingSuite) TearDownTest(c *gc.C) {
-	s.close()
+	if s.close != nil {
```
Comment (rogpeppe) [inline store/testing/meeting.go]: When might this be nil?

File: store/testing/store.go [hunk 2]
```diff
@@ -41,7 +41,9 @@ func (s *StoreSuite) SetUpTest(c *gc.C) {
 }
 
 func (s *StoreSuite) TearDownTest(c *gc.C) {
-	s.close()
+	if s.close != nil {
```
Comment (rogpeppe) [inline store/testing/store.go]: Similar here.

File: store/mgostore/bakery_test.go [hunk 3]
```diff
@@ -4,22 +4,39 @@
...
-	testing.IsolatedMgoSuite
+	db *mgotest.Database
 }
 
 var _ = gc.Suite(&bakerySuite{})
 
+func (s *bakerySuite) SetUpTest(c *gc.C) {
+	var err error
+	s.db, err = mgotest.New()
+	c.Logf("ERR: %#v", err)
```
Comment (rogpeppe) [inline store/mgostore/bakery_test.go]: debugging remnant?

File: store/testing/meeting.go (hunk 1 above)
Comment (mhilton) [reply store/testing/meeting.go]: When it wasn't set up because the database connection failed.

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.theblues.io:8080/job/candid-merge/64

**Pull request merged**